streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -90,24 +88,38 @@ public void before() {
         streams = new KafkaStreams(builder.build(), props);
     }
 
+    @After
+    public void cleanup() {
+        if (streams != null) {
```

Comment:
minor: this variable is not introduced in this PR but it may be good to rename this to `sharedStreams` or any other better name as `streams` instance is created locally in some of the tests.